gcloud-java-examples/src/main/java/com/google/gcloud/examples/BigQueryExample.java

```diff
@@ -570,15 +570,15 @@ protected String params() {
    */
   private static class ExtractAction extends JobRunAction {
     @Override
-    ExtractJobInfo parse(String... args) throws Exception {
+    JobInfo parse(String... args) throws Exception {
       if (args.length >= 4) {
         String dataset = args[0];
         String table = args[1];
         String format = args[2];
         TableId tableId = TableId.of(dataset, table);
-        return ExtractJobInfo.builder(tableId, Arrays.asList(args).subList(3, args.length))
-            .format(format)
-            .build();
+        ExtractJobConfiguration configuration =
+            ExtractJobConfiguration.of(tableId, Arrays.asList(args).subList(3, args.length));
```

Comment:
`ExtractJobConfiguration.of(tableId, Arrays.asList(args).subList(3, args.length), format);` ?